river/tree/hoeffding_tree_regressor.py

```diff
@@ -44,6 +44,21 @@ class HoeffdingTreeRegressor(BaseHoeffdingTree, base.Regressor):
     nominal_attributes
         List of Nominal attributes identifiers. If empty, then assume that all numeric attributes
         should be treated as continuous.
+    attribute_observer
```

Comment:
:-)

Reply:
Updated in commit [f1eab02](https://github.com/creme-ml/creme/pull/379/commits/f1eab0274d21fedd3f599f8597e6a2f0230cbe26).